data/json/items/book/cooking.json

```diff
@@ -171,34 +173,34 @@
     "skill": "cooking",
     "max_level": 1,
     "intelligence": 5,
-    "time": "24 m",
+    "time": "23 m",
```

Comment:
Why the change in reading time?

Reply:
arbitrary! this change of a minute less in time to read is an unnecessary distraction, it ought not to have been included!!